tests/foreman/ui/test_dashboard.py

```diff
@@ -196,7 +196,10 @@ def test_positive_task_status(session):
         session.dashboard.action({
             'LatestFailedTasks': {'name': 'Synchronize'}
         })
-        values = session.task.read('Synchronize')
+        values = session.task.read(
+            "Synchronize repository '{}'; product '{}'; organization '{}'"
+            .format(repo.name, product.name, org.name)
```

Comment:
Since this is exactly the same as string used couple of lines above, could you place it in new variable?
```python
task_name = "Synchronize repository '{}'; product '{}'; organization '{}'"
￼	     .format(repo.name, product.name, org.name)
...
assert tasks['table'][0]['Action'] == task_name
...
values = session.task.read(task_name)
```